astoria/managers/astdiskd/static.py

```diff
@@ -0,0 +1,124 @@
+"""Allows disks to be added manually through local filesystem paths."""
+import logging
+from typing import TYPE_CHECKING, Callable, Coroutine
+
+from astoria.common.manager_requests import (
+    AddStaticDiskRequest,
+    RemoveAllStaticDisksRequest,
+    RemoveStaticDiskRequest,
+    RequestResponse,
+)
+from astoria.common.messages.astdiskd import DiskUUID
+
+from .disk_provider import DiskProvider
+
+LOGGER = logging.getLogger(__name__)
+
+if TYPE_CHECKING:
+    from . import DiskManager
```

Comment:
nit: Importing from the file rather than the parent module makes it clearer where the class comes from :)

```suggestion
    from .disk_manager import DiskManager
```